command/agent/config/config.go

```diff
@@ -340,6 +341,16 @@ func parseMethod(result *Config, list *ast.ObjectList) error {
 		m.WrapTTLRaw = nil
 	}
 
+	if m.Namespace != "" {
```

Comment:
You can use `namespace.Canonicalize(m.Namespace)` in the helper/namespace package for this.

Reply:
Thanks!